uwblueprint/ccbc
# [Tech] Frontend Global Toasts

## Notion ticket link

[Frontend Toasts](https://www.notion.so/uwblueprintexecs/Frontend-Toasts-d85da818e9a7440db41aff7888cebe7f)



## Implementation description
* created a central component that can be called from anywhere to display a chakra ui toast.



## Steps to test
1. Publish a review (verify a toast appears on the bottom right of the screen displaying a success message)
2. Edit a review (verify a toast appears on the bottom right of the screen displaying a success message)


## What should reviewers focus on?
* Time a toast should remain on screen, previously it was 10 seconds which imo feels extreme, it is currently 5 seconds as of this PR
* Info vs Success for publishing/editing reviews? It was previously info 
![image](https://user-images.githubusercontent.com/37082847/172035681-a2856dcd-206b-4e82-9724-aa4b80b12db5.png)
![image](https://user-images.githubusercontent.com/37082847/172035667-b12c040e-0412-4e79-b70c-d13e81ee5c60.png)
* Here is an example of what happens when we have multiple toasts on screen (
![image](https://user-images.githubusercontent.com/37082847/172036347-932ce497-e40a-4352-9fe6-4e79b63f1152.png))
the 2nd toast is error just to be able to differentiate between the 2 (in case they overwrote each other). 
* I have also tested navigating to another page while a toast is visible and the toast remains on it's page until it's specified timeout


## Checklist
